repair/row_level.cc

```diff
@@ -524,14 +524,14 @@ class repair_writer : public enable_lw_shared_from_this<repair_writer> {
     uint64_t _estimated_partitions;
     size_t _nr_peer_nodes;
     // Needs more than one for repair master
```

Comment:
This comment is no longer correct? (this patch changes the vector to a single one)

Reply:
Yes. We should drop this comment. Thanks.